security-indicators/base/openshift-templates/security-indicators.yaml

```diff
@@ -85,7 +87,6 @@ objects:
                 - 900
               value: "{{workflow.duration}}"
 
-      onExit: exit-handler
```

Comment:
isn't this required to remove the PVC? removing the pod is going to delete also the `PersistentVolumeClaim` object?

Reply:
Deleting the pod will also delete its log, do we need that log?

Reply:
> isn't this required to remove the PVC? removing the pod is going to delete also the `PersistentVolumeClaim` object?

That's a question about Argo Workflows behavior, for OpenShift only the answer is 'no'

Reply:
That is what I thought and I think argo cannot do that, podGC is for pods not for PVC.

Reply:
> Deleting the pod will also delete its log, do we need that log?

We already keep the life of workflow to 25 seconds in middletier, the logs should be stored on Ceph if we need them, we need to set that in the workflow-controller CM

Reply:
It's not required to remove the PVC, all that is required is that the pods be deleted

Reply:
and who removes the PVC requested? we need to free it

Reply:
It is removed automatically once all pods have been deleted

Reply:
I didn't know, then I trust you :)

Reply:
I think it is deleted based on the owner reference configured by argo.

Speaking of this, It would be great to check the reclaim policy for volumes in the cluster - for some configuration, we can eat up all the available volumes if they are not recycled. I believe we are not affected, we would already encounter this with the number of workflows we processed.